applications/video_streaming_demo_enhanced/CMakeLists.txt

```diff
@@ -0,0 +1,236 @@
+             echo 'Starting streaming client...'
+             ./applications/video_streaming_demo_enhanced/video_streaming_demo_client/cpp/streaming_client_demo_enhanced --config applications/video_streaming_demo_enhanced/video_streaming_demo_client/cpp/streaming_client_demo_replayer.yaml > \$CLIENT_LOG 2>&1 &
+             CLIENT_PID=\$!
+
+             # Let streaming run for 30 seconds
+             echo 'Letting streaming run for 30 seconds...'
+             sleep 30
+
+             # Stop client and server gracefully
+             echo 'Stopping client...'
+             kill -TERM \$CLIENT_PID 2>/dev/null || true
+             sleep 2
+             kill -KILL \$CLIENT_PID 2>/dev/null || true
+
+             echo 'Stopping server...'
+             kill -TERM \$SERVER_PID 2>/dev/null || true
+             sleep 2
+             kill -KILL \$SERVER_PID 2>/dev/null || true
+
+             # Now verify logs for actual streaming behavior
+             echo ''
+             echo '=== Verifying Server Logs ==='
+             
+             # Check for client connection
+             if grep -q 'Client connected' \$SERVER_LOG; then
+               echo '✓ Server: Client connected'
+               SERVER_SUCCESS=\$((SERVER_SUCCESS + 1))
+             else
+               echo '✗ Server: No client connection detected'
+             fi
+             
+             # Check for upstream connection
+             if grep -q 'Upstream connection established' \$SERVER_LOG; then
+               echo '✓ Server: Upstream connection established'
+               SERVER_SUCCESS=\$((SERVER_SUCCESS + 1))
+             else
+               echo '✗ Server: Upstream connection not established'
+             fi
+             
+             # Check for downstream connection
+             if grep -q 'Downstream connection established' \$SERVER_LOG; then
+               echo '✓ Server: Downstream connection established'
+               SERVER_SUCCESS=\$((SERVER_SUCCESS + 1))
+             else
+               echo '✗ Server: Downstream connection not established'
+             fi
+             
+             # Check for frame processing
+             if grep -q 'Processing UNIQUE frame' \$SERVER_LOG; then
+               FRAME_COUNT=\$(grep -c 'Processing UNIQUE frame' \$SERVER_LOG)
+               echo \"✓ Server: Processed \$FRAME_COUNT unique frames\"
+               SERVER_SUCCESS=\$((SERVER_SUCCESS + 1))
+             else
+               echo '✗ Server: No frames processed'
+             fi
+             
+             # Check for frame stats (optional but good indicator)
+             if grep -q 'Frame Processing Stats' \$SERVER_LOG; then
+               echo '✓ Server: Frame processing statistics logged'
+               SERVER_SUCCESS=\$((SERVER_SUCCESS + 1))
+             fi
+
+             echo ''
+             echo '=== Verifying Client Logs ==='
+             
+             # Check for successful frame sending
```

Comment:
Thank you for adding regex checks. It looks like the test is still missing regex coverage for client frame reception, would you please add that as well?